nixos/tests/systemd-networkd-dhcpserver.nix

```diff
@@ -28,10 +28,12 @@ import ./make-test-python.nix ({pkgs, ...}: {
               DHCPServer = true;
               Address = "10.0.0.1/24";
             };
-            dhcpServerConfig = {
-              PoolOffset = 100;
-              PoolSize = 1;
-            };
```

Comment:
This would remove the testing of dynamic leases.

It might be worth keeping this dynamic lease test, and creating a separate test for static leases?

Reply:
Agreed, that makes sense. At the moment, however, it doesn't work at all 😐.

Reply:
I moved the test to a new file.